tests/syntax_tests/data/parsing/errors/typeDef/expected/nestedRecord.res.txt

```diff
@@ -0,0 +1,21 @@
+
+  Syntax error!
+  syntax_tests/data/parsing/errors/typeDef/nestedRecord.res:3:11-47
```

Comment:
Not sure why this has been added. Any idea @zth ?

Reply:
I think you can exclude this change @MiryangJung 

It should be resolved in the master branch